Sources/SUITCase/SUITCase+verifyScreenshot.swift

```diff
@@ -101,7 +101,7 @@ extension SUITCase {
                                   withThreshold customThreshold: Double? = nil,
                                   withMethod method: SUITCaseMethod = SUITCaseMethodWithTolerance(),
                                   withLabel label: String? = nil) throws {
-        guard UIDevice.isSimulator else {
```

Comment:
Code might be much cleaner, if you switch back to the guard and apply the second de Morgan's rule